__tests__/__fixtures__/exampleBls12381JwkKeyPair.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "did:example:489398593#test",
+  "controller": "did:example:489398593",
+  "publicKeyJwk": {
+    "kid": "did:example:489398593#test",
+    "crv": "BLS12-381_G2",
+    "kty": "EC",
```

Comment:
Maybe just use all underscores to avoid confusion

Reply:
seems like a good suggestion. @tplooker this is a bit different from the example I went off [here](https://github.com/decentralized-identity/sidetree/issues/588#issuecomment-643059644).

Would you be alright to change this to `BLS12_381_G2` and `BLS12_381_G1` for the G1 representation?

Reply:
Id say lets change to BLS12381_G1 and BLS12381_G2

Reply:
Updated to be in the latest commit.